File: platform-core-service/platform-core-service.Common/Models/DTOs/EntityDTO/CommunityCommentsReport/SelectCommunityCommentsReportDTO.cs
using platform_core_service.Common.Models.DTOs.EntityDTO.CommunityContentReport;

namespace platform_core_service.Common.Models.DTOs.EntityDTO.CommunityCommentsReport
{
    public class SelectCommunityCommentsReportDTO : BaseSelectCommunityReportDTO
    {
        public string CommentId { get; set; } = null!;
        public SelectReportedCommentDTO? Comment { get; set; }
    }
}

File: frontend/src/types/community-comments-report/select-community-comments-report-dto.ts
import { BaseSelectCommunityReportDTO } from "../community-content-report/base-select-community-report-dto";
import { SelectReportedCommentDTO } from "../community-content-report/select-reported-content-dto";

export interface SelectCommunityCommentsReportDTO extends BaseSelectCommunityReportDTO {
    commentId: string;
    comment?: SelectReportedCommentDTO | null;
}
